docs/contents/dev-docs/setup.md

```diff
@@ -78,3 +150,7 @@ source repository and there open the pull request as depicted below:
 We monitor the GitHub repo and try to follow up with comments within a working
 day.
 
+## Next Steps
+
+After getting familiar with the various ACK source code repositories, now learn
+[how to build an ACK service controller](../build-controller).
```

Comment:
This and at other places where `.md` files are referenced:
* should `./` be used instead of `../`?
* should full file name (including `.md` extension) be used instead?

like: `./building-controller.md`

Reply:
it's this way because of the way that the doc site generator works... I think once we do the repo transfer/move, we should take a look at tweaking our doc generator.